hack/jenkins/test-dockerized.sh

```diff
@@ -55,5 +55,10 @@ go install ./cmd/...
 ./hack/install-etcd.sh
 
 make test-cmd
+
+export STORE_KUBE_TIMEOUT="${KUBE_TIMEOUT}"
+export KUBE_TIMEOUT='-timeout 600s'
```

Comment:
This is already the default, isn't it?

https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test-integration.sh#L34

I'm pretty sure noting else in here is using KUBE_TIMEOUT either so no reason to store it. Why don't you just remove  line 47?

Reply:
"make test-integration" and "make test" are the only two places that use it. I guess that it would make sense to remove that line, but only if make test isn't called by any of the code in here